lib/razor/cli/command.rb

```diff
@@ -41,8 +41,10 @@ def extract_command
     body = {}
     until @segments.empty?
       argument = @segments.shift
-      if argument =~ /\A--([a-z-]+)(=(.+))?\Z/
-        # `--arg=value` or `--arg value`
+      if argument =~ /\A--([a-z-]{2,})(=(.+))?\Z/ or
+          argument =~ /\A-([a-z])(=(.+))?\Z/
+        # `--arg=value`/`--arg value`
+        # `-a=value`/`-a value`
```

Comment:
There is some use of the `OptionParser` library in Razor, but it's currently only used statically for top-level `razor` commands, things like RAZOR_API or verbose mode. I agree it would be a great refactoring to convert this into dynamically generated `OptionParser` instances.

I guess the only reason not to support combining of short flags is if the user intends to use long-form and omits a dash, e.g. `-name` instead of `--name`, which could produce a confusing error message. I'm fine with supporting that style, either way.